nano/templates/ship_core.tmpl

```diff
@@ -0,0 +1,120 @@
+{{if data.error}}
+	<h2>An error has occured:</h2>
+	Additional information: {{:data.error}}<br>
+	<i>Please try again. If the problem persists contact your system administrator for assistance.</i>
+	<div class="item">
+		{{:helper.link('Refresh', null, { 'refresh' : 1 })}}
+	</div>
+{{else}}
+	<div class="itemLabel">
```

Comment:
Replace this with network_settings as a UI helper. Check out fabricator.tmpl for an example.